### DISCUSSION TITLE
Question: How to modify the position of backbottom button.

### DISCUSSION THREAD

**Discussion by CSLukkun** [ant-design/pro-chat#68] [Q&A] [answered]
Thanks for your attention.

I want to modify the position of backbotton button instead of `16px` in the source code.

I don't want to modify the content of the source package. 

Could u provide me with a good solution?

Thank u again.

**Comment by CSLukkun** [comment]
```
   return (
      <Button
        className={cx(styles, className)}
        icon={<Icon icon={ListEnd} />}
        onClick={scrollToBottom}
        ref={ref}
        size={'small'}
        style={{ bottom: 16, position: 'absolute', right: 16, ...style }}
      >
        {text || 'Back to bottom'}
      </Button>
    );
```
The code comes from the source code. 
The bottom has been fixed. I want to modify that.

**Comment by ONLY-yours** [comment]
sounds like we should provide such a api 

```js
backBottomConfig ={ 
   title: string,
   onCallBack:()=>{},
   className: string,
   styles: Css,
   render: (defaultDom, scrollToBottom, backBottomConfig)=> ReactNode,
   alwaysShow: boolean
}
```

**Comment by ONLY-yours** [comment]
are you willing to help us add this api ?

**Comment by ONLY-yours** [comment, marked as answer]
now u can use this to modify it (more detial see https://github.com/ant-design/pro-chat/pull/69)

```
       <ProChat
          backtoBottomConfig={{
             style:{
                // any css u like
             }
          }}
        />
```

className is also work , also u can use render to custom
